scripts/apps/search/components/Item.tsx

```diff
@@ -134,8 +138,14 @@ export class Item extends React.Component<IProps, IState> {
     }
 
     select(event) {
-        if (!this.props.item.gone) {
-            this.props.onSelect(this.props.item, event);
+        const {$timeout} = this.props.svc;
+
+        if (!this.props.item.gone && !this.clickTimeout) {
```

Comment:
What is `item.gone`?

Reply:
Shouldn't the React component unmount if it's spiked/sent to another stage?

Reply:
it's visible but grayed out so other users can see that the item was there but is not there anymore

Reply:
Got it.